web/pages/embed/[username]/[contractSlug].tsx

```diff
@@ -74,66 +78,87 @@ export function ContractEmbed(props: { contract: Contract; bets: Bet[] }) {
     creatorId: contract.creatorId,
   })
 
+  const { creatorName, creatorUsername, creatorId, creatorAvatarUrl } = contract
+  const { resolvedDate } = contractMetrics(contract)
   const isBinary = outcomeType === 'BINARY'
   const isPseudoNumeric = outcomeType === 'PSEUDO_NUMERIC'
 
   const href = `https://${DOMAIN}${contractPath(contract)}`
 
-  const { setElem, height: graphHeight } = useMeasureSize()
+  const { setElem, width: graphWidth, height: graphHeight } = useMeasureSize()
 
   const [betPanelOpen, setBetPanelOpen] = useState(false)
 
   const [probAfter, setProbAfter] = useState<number>()
 
-  return (
-    <Col className="h-[100vh] w-full bg-white">
-      <Row className="justify-between gap-4 px-2">
-        <div className="text-xl text-indigo-700 md:text-2xl">
-          <SiteLink href={href}>{question}</SiteLink>
+  const user = useUser()
+  if (user && (isBinary || isPseudoNumeric)) {
```

Comment:
What if the market is closed or resolved? You would probably want the `else` case for that.